site/docs/guidelines/data-visualization.mdx

```diff
@@ -0,0 +1,157 @@
+    - Adjust font size according to bubble size. Again, you might show large text only on the bubbles that tell the story.
+- Use a "line of best fit" or a "trend line" to **surface trends** in the data.
+
+Learn more about scatterplots:
+
+- [Data Viz Catalogue: Scatterplot](https://datavizcatalogue.com/methods/scatterplot.html)
+- [Data Viz Project: Scatterplot](https://datavizproject.com/data-type/scatter-plot/)
+- [Morningstar: scatter plot](https://designsystem.morningstar.com/charts/scatter-plot.html)
+- [Government of Canada, Aurora: Scatterplots](https://design.gccollab.ca/data/scatterplots)
+- [Spectrum: scatter plot](https://spectrum.adobe.com/page/scatter-plot/)
+- [Predix: scatter](https://www.predix-ui.com/#/elements/vis/px-vis-framework/px-vis-scatter)
+
+Learn more about bubble charts:
+
+- [The Data Viz Project: Bubble Chart](https://datavizproject.com/data-type/bubble-chart/)
+- [The Data Visualisation Catalogue: Bubble Chart](https://datavizcatalogue.com/methods/bubble_chart.html)
+- [Carbon: Bubble Chart](https://www.carbondesignsystem.com/data-visualization/basic-charts#bubble)
+- [Morningstar bubble chart](https://designsystem.morningstar.com/legacy/charts/bubble.html)
+- [IBM: Bubble chart](https://www.ibm.com/design/v1/language/experience/data-visualization/chart-models/bubblechart/)
+- [US Gov: Bubble Chart](https://xdgov.github.io/data-design-standards/visualizations/bubble-chart)
+- [AntV: G2: Bubble](https://g2.antv.vision/en/examples/point/bubble#bubble)
+
+## Bar charts
+
+Bar charts are used to compare discrete, numeric data across categories.
+
+- Order the bars. For example, you might use natural data order, such as months of the year, or rank them, ordering the bars from shortest to longest.
+- For long bar labels, consider using a horizontal bar chart or presenting the label on the bar.
+
+We often used a stacked bar to represent likert scales. Learn more in [Visualizing Likert Scale Data: Same data, displayed seven different ways by Alana Pirrone](https://medium.com/nightingale/seven-different-ways-to-display-likert-scale-data-d0c1c9a9ad59?source=friends_link&sk=60cb93604b71ecc8820cc785ed1afd1a).
+
+Learn more about bar charts:
+
+- [Data viz project: Bar Chart (Horizontal)](https://datavizproject.com/data-type/bar-chart-horizontal/)
+- [Data viz project: Stacked Bar Chart](https://datavizproject.com/data-type/stacked-bar-chart/)
+- [The Data Visualisation Catalogue: Bar Chart](https://datavizcatalogue.com/methods/bar_chart.html)
+- [Shopify Polaris: horizontal bar charts](https://polaris.shopify.com/design/data-visualizations#section-horizontal-bar-charts)
+- [Shopify Polaris: vertical column charts](https://polaris.shopify.com/design/data-visualizations#section-vertical-column-charts)
+- [Morningstar: horizontal bar chart](https://designsystem.morningstar.com/charts/horizontal-bar.html)
+- [Government of Canada, Aurora: Bar charts](https://design.gccollab.ca/data/column-and-bar-graphs)
+
+## Hero data
```

Comment:
I think this one needs an example. I know the concept of heroing data, but not the noun "hero data"

Reply:
Updated explanation:

> Hero data refers to prominent figures on a page that draw attention to individual values using proportionally larger type sizes. Use hero data to highlight a single value when it does not need context or comparisons.

… and a picture!